### PR TITLE
Put Many, Clean up

### PR DESCRIPTION
**Pull request by xeust**

### CODE REVIEW

File: docs/Lib.md [hunk 1]
```diff
@@ -401,15 +401,135 @@ Returns the item on a successful insert, and throws an error if the key already
...
+    { label: 'Python', value: 'py', },
+  ]
+}>
+<TabItem value="js">
+
+**`async putMany(data)`**
+
+#### Parameters
+
+- **`data`** (required) – Accepts: `list` of `items`, where each `item` can be an `object` (serializable), `string`, `number`, `boolean` or `array`.
```
Comment (abdelhai) [inline docs/Lib.md]:
```suggestion
**`async putMany(items)`**

#### Parameters

- **`items`** (required) – Accepts: `Array` of items, where each "item" can be an `object` (serializable), `string`, `number`, `boolean` or `array`.
```

File: docs/Lib.md [hunk 2]
```diff
@@ -401,15 +401,135 @@ Returns the item on a successful insert, and throws an error if the key already
...
+
+</TabItem>
+<TabItem value="py">
+
+**`put_many(data):`**
+
+#### Parameters
+
+- **`data`** (required) – Accepts: `list` of `items`, where each `item` can be an `object` (serializable), `string`, `number`, `boolean` or `array`.
+    - Description: The list of items to be stored.
```
Comment (abdelhai) [inline docs/Lib.md]:
```suggestion
**`put_many(items):`**

#### Parameters

- **`data`** (required) – Accepts: `list` of items, where each "item" can be an `dict` (JSON serializable), `str`, `int`, `bool`, `float` or `list`.
    - Description: The list of items to be stored.
```

File: docs/Lib.md [hunk 3]
```diff
@@ -501,39 +623,38 @@ const mySecondSet = await db.fetch([
...
+    bar(subArray)
 
 }
 ```
 </TabItem>
 
 <TabItem value="py">
 
-**`fetch(query=None, buffer=None, limit=None):`**
+**`fetch(query=None, limit=None, buffer=None):`**
```
Comment (abdelhai) [inline docs/Lib.md]:
```suggestion
**`fetch(query=None, limit=2000, buffer=None):`**
```

File: docs/Lib.md [hunk 4]
```diff
@@ -575,21 +696,21 @@ my_second_set = db.fetch([{"age?lt": 30}, {"hometown": "Greenville"}])
...
-Iterating through the generator yields lists containing items, each list of max length `buffer`.
-
+Iterating through the generator yields lists containing objects, each list of max length `buffer`.
 
 
 #### Example using buffer, limit
 
 ```py
-items = db.fetch(my_query, buffer=10, limit=100) # items is up to the limit length, 100
+def foo(my_query, bar):
```
Comment (abdelhai) [inline docs/Lib.md:709]: why put inside a function?
also, what does `bar` do?

**Review by abdelhai — CHANGES_REQUESTED** [review]

File: docs/Lib.md (hunk 4 above)
Comment (xeust) [reply docs/Lib.md:709]: Intention was to show how the results of a query can be operated on.

Nonetheless, I think it should be consistent across JS and Python.

`bar` is some arbitrary function. Identity of `bar` is unimportant, meant to demonstrate how the query can be operated on. (following convention foo, bar, baz https://en.wikipedia.org/wiki/Metasyntactic_variable).

**Review by abdelhai — APPROVED** [review]

**Pull request merged**